doc/dialects/mysql.md

```diff
@@ -1,129 +0,0 @@
-# MySQL SQL Dialect
-
-[MySQL](https://www.mysql.com/) is an open-source relational database management system.
```

Comment:
Add a link to the new dialect instead. Some websites could still point on that page